src/main/java/mate/academy/dao/impl/CinemaHallDao.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.dao.impl;
```

Comment:
The package name `mate.academy.dao.impl` suggests that this file contains an implementation, but it actually defines an interface. Consider renaming the package to `mate.academy.dao` to better reflect its contents.